scripts/prepdocs.ps1

```diff
@@ -33,5 +33,6 @@ $argumentList = "./scripts/prepdocs.py `"$cwd/data/*`" $adlsGen2StorageAccountAr
 "--openaiservice `"$env:AZURE_OPENAI_SERVICE`" --openaikey `"$env:OPENAI_API_KEY`" " + `
 "--openaiorg `"$env:OPENAI_ORGANIZATION`" --openaideployment `"$env:AZURE_OPENAI_EMB_DEPLOYMENT`" " + `
 "--openaimodelname `"$env:AZURE_OPENAI_EMB_MODEL_NAME`" --index $env:AZURE_SEARCH_INDEX " + `
-"--formrecognizerservice $env:AZURE_FORMRECOGNIZER_SERVICE --tenantid $env:AZURE_TENANT_ID -v"
+"--formrecognizerservice $env:AZURE_FORMRECOGNIZER_SERVICE --tenantid $env:AZURE_TENANT_ID" + `
+"--openaihost $env:OPENAI_HOST --openaikey $env:OPENAI_API_KEY -v"
```

Comment:
I see host and key on lines 32-33 so I think they didnt get dropped from this one?

Reply:
My mistake. You’re spot on. Only the `prepdocs.sh` file needs to be changed. Just removed the unnecessary args from the Powershell script.

Reply:
I am confused between OpenAI Orgnization and openaihost. Could you please help me how to pass these details-
OPENAI_API_TYPE = "azure"
OPENAI_API_VERSION = "2023-07-01-preview"
OPENAI_API_KEY = "xxxxxxx"
OPENAI_API_BASE = " https://xxxxxx.openai.azure.com/"

Reply:
Replied to your comment on other thread